presto-blackhole/src/main/java/com/facebook/presto/plugin/blackhole/BlackHoleConnector.java

```diff
@@ -24,6 +24,7 @@
 import com.facebook.presto.spi.session.PropertyMetadata;
 import com.facebook.presto.spi.transaction.IsolationLevel;
 import com.facebook.presto.spi.type.TypeManager;
+import com.facebook.presto.spi.type.TypeSignatureParameter;
```

Comment:
Extend commit message to explain what is done and why. And change TypeRegistry to TypeManager in commit message.